### PR TITLE
data:BMayowa

### PR DESCRIPTION
**Pull request by BMayowa**
Mayowa Obisesan's LinkFree Profile

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3513"></a>

### CODE REVIEW

File: data/BMayowa.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        {
+            "name": "GitHub",
+            "url": "https://github.com/BMayowa",
+            "icon": "FaGithub"
+        },
+        {
+            "name": "LinkedIn",
+            "url": "https://linkedin.com/in/mayowa-obisesan",
+            "icon": "FaLinkedin"
+        },
```
Comment (Tejender1521) [inline data/BMayowa.json:42]: Remove this ","

**Review by Tejender1521 — CHANGES_REQUESTED** [review]
Validate JSON using online tools to find errors in your json file

**Review by senali-d — CHANGES_REQUESTED** [review]
It is better to add an avatar. You can use same profile image in the github as follows.

`https://www.github.com/BMayowa.png`

Feel free to ask if you have any doubts.

File: data/BMayowa.json (hunk 1 above)
Comment (BMayowa) [reply data/BMayowa.json]: Thanks. Done

File: data/BMayowa.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    "name": "Mayowa Obisesan",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Software Engineer | Built [createsummary.com](https://www.createsummary.com) | [Eye CSS](https://github.com/BMayowa/EYE_CSS)",
+    "tags": [
+        "Python",
+        "JavaScript",
+        "Django",
+        "FastAPI",
+        "Vue JS"
```
Comment (senali-d) [inline data/BMayowa.json:11]: Missing `,`
```suggestion
        "Vue JS",
```

File: data/BMayowa.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        "NodeJS"
```
Comment (senali-d) [inline data/BMayowa.json:12]: Missing `,`
```suggestion
        "NodeJS",
```

File: data/BMayowa.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        "CSS"
```
Comment (senali-d) [inline data/BMayowa.json:13]: Missing `,`
```suggestion
        "CSS",
```

**Review by senali-d — CHANGES_REQUESTED** [review]

File: data/BMayowa.json [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+		{
+			"name": "LinkedIn",
+			"url": "https://linkedin.com/in/mayowa-obisesan",
+			"icon": "FaLinkedin"
+		}
+	],
+	"milestones": [{
+			"title": "Released Summary",
+			"date": "November 30, 2022",
+			"icon": "",
```
Comment (Panquesito7) [inline data/BMayowa.json:51]: I think you should add an icon. If you're not going to use an icon, remove the field, then.

Comment (SaraJaoude) [reply data/BMayowa.json:51]: You make a valid point, but this can come in a future PR

**Pull request merged**